man/systemd.journal-fields.xml

```diff
@@ -56,10 +56,9 @@
     their syntax but with fields that can include binary data.
     Primarily, fields are formatted UTF-8 text strings, and binary
     formatting is used only where formatting as UTF-8 text strings
-    makes little sense. New fields may freely be defined by
-    applications, but a few fields have special meaning. All fields
-    with special meanings are optional. In some cases, fields may
-    appear more than once per entry.</para>
+    makes little sense. New fields may freely be defined (multiple times)
```

Comment:
"may freely be defined" doesn't sound good. It was already there, but maybe you can change it while working on this para. Maybe "Applications can attach arbitrary fields, and those fields can occur more than once"?